releasenotes/_deconst.json

```diff
@@ -0,0 +1,8 @@
+{
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13”,
+    "githubUrl": "https://github.com/rackerlabs/docs-rackspace/",
```

Comment:
Perhaps it should be docs-rpc, so there is a central point for RCPO docs related issues?